.github/workflows/publish-model-2-packages.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish org.lflang to GitHub Packages
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+      - name: Prepare build environment
+        uses: ./.github/actions/prepare-build-env
+      - name: Publish package
+        uses: gradle/gradle-build-action@0d13054264b0bb894ded474f08ebb30921341cee
+        with:
+          arguments: org.lflang:publish
```

Comment:
What gets published as a result of this action, exactly? Is there a way to do a dry run and find out?

Reply:
I am not sure if there is a way to do a dry-run of this, sadly. I know a tool that can simulate all github actions offline. But I guess we don't get the GITHUB_TOKEN that is injected in the containers that run all actions. 

In theory this this command would publish the `jar` that is generated from the `org.lflang` submodule of the repo. It might need some adjustment when it comes to the dependencies of the MWE2 plugins are doing them proper, but I guess it is not much extra work! 

In any case, I think the only way to see if this is working is to get the developer machine (you, @lhstrh?) and try to upload the jars to GH packages. This is the default flow for Maven Central so I am not sure if it can be replicated. I will try to check it.

Reply:
OK, thanks! I'll tinker with this.